MapboxGeocoder/MBGeocodeOptions.swift

```diff
@@ -46,6 +46,11 @@ public class GeocodeOptions: NSObject {
      */
     public var allowedRegion: RectangularRegion?
     
+    /**
+     Limit the number of results returned. The default is `5` for forward geocoding and `1` for reverse geocoding.
+     */
+    public var maximumResultCount: UInt = 0
```

Comment:
If all the initializers set this property, it doesn't need a default value of 0 here (which would be misleading anyways).